ui/public/locales/en.json

```
@@ -2164,6 +2180,13 @@
 "label.system.vm.scaled.up": "System VM Scaled Up",
 "label.system.vms": "System VMs",
 "label.system.wide.capacity": "System-wide capacity",
+"label.systemcycleusage": "User-, System and Idle Cycles",
+"label.systemloadaverages": "1, 5 and 15 mintute load averages",
+"label.systemmemoryfree": "Free System Memory",
+"label.systemmemorytotal": "Total System Memory",
+"label.systemmemoryused": "Used System Memory",
+"label.systemmemoryvirtualsize": "Virtual Process Size (when all shared libraries are loaded)",
```

Better the labels simple in UI, you can add '_when all shared libraries are loaded_' , and more details in the documentation.

```suggestion
"label.systemmemoryvirtualsize": "Virtual Process Size",
```

---

how about
```suggestion
"label.systemmemoryvirtualsize": "Virtual Process Total Size",
```
?